collect_app/src/main/java/org/odk/collect/android/preferences/screens/BaseAdminPreferencesFragment.java

```diff
@@ -17,6 +18,9 @@
     @Named("ADMIN_SETTINGS_STORE")
     SettingsStore adminSettingsStore;
 
+    @Inject
+    CurrentProjectProvider currentProjectProvider;
```

Comment:
You can move it one level up to `BasePreferencesFragment` since it's used in both `BaseAdminPreferencesFragment` and `BaseGeneralPreferencesFragment`.